src/app/[locale]/_component/accordion/faqDefault.tsx

```diff
@@ -29,7 +29,7 @@ export const FAQ = ({ flow = Flows.DEFAULT }: FAQProps) => {
       details: t('firstresponse'),
     },
     {
-      summary: t('fourthquestion'),
+      summary: t('fouthquestion'),
```

Comment:
sorry, its my typo, can you replace in all project with "fourthquestion" (too in json, i replace too in ditto!)?